jdaviz/core/template_mixin.py

```diff
@@ -4243,7 +4243,13 @@ def auto(self):
         Access the ``auto`` property of the ``AutoTextField`` object.  If enabling, the ``label``
         will automatically be changed and kept in sync with the default label.
         """
-        return self.auto_label.auto
+        # TODO: remove when long-term fix is implemented
+        try:
+            # This line is causing a massive traceback in CI when run on python 3.11,
+            # temporarily wrap in try/except to get CI passing
+            return self.auto_label.auto
+        except AttributeError:
+            return ""
```

Comment:
this could cause some truthy-issues elsewhere (this will effectively act as `False`), but I agree this is the least obtrusive workaround while we work on the eventual fix.  Works for me if CI passes - thanks!